README.md

```diff
@@ -30,25 +32,29 @@
 | `-s` or `--size`           | n/a                                             | Show file size in normal format                                 |
 | `-S`                       | n/a                                             | Sort by size (largest first)                                    |
 | `-R` or `--revcursive`     | n/a                                             | Show files in sub-directories recursively (not implemented yet) |
- | `-nc` or `--no-icons`      | n/a                                             | Don't show icons                                                |
+| `-nc` or `--no-icons`      | n/a                                             | Don't show icons                                                |
 | `-ic` or `--icons-color`   | Icon color in hex format or Auto for auto color | Use custom icons color (not implemented yet)                    |
- | `-tc` or `--text-color`    | Text color in hex format or Auto for auto color | Use custom text color  (not implemented yet)                    |
+| `-tc` or `--text-color`    | Text color in hex format or Auto for auto color | Use custom text color (not implemented yet)                     |
 | `--no-owner`               | n/a                                             | Don't show owner name in long format                            |
 | `--no-colors`              | n/a                                             | Don't use colors                                                |
 | `-nd` or `--no-date`       | n/a                                             | Don't show last modified date                                   |
 | `-nn` or `--no-name`       | n/a                                             | Don't show file name                                            |
 | `-L`                       | n/a                                             | Sort by last modified date (latest first)                       |
 
-
-
 ## Requirements for development:
+
 - [nerd-font](https://www.nerdfonts.com)
 - Gradle 7.4
 - jdk 17
 - IntelliJ IDEA (not required but recommended)
 
+## TODO
+
+- [ ] Add Tests
+- [ ] Add Formatter
```

Comment:
I mean code formatter like prettier

It would be nice if you added a pre-commit hook that takes care of formatting and testing before commiting/pushing to the master branch.